f/zeus/fn_zeusInit.sqf

```diff
@@ -66,23 +79,28 @@ _unit assignCurator _curator;
 _curator setCuratorWaypointCost 0;
 {_curator setCuratorCoef [_x,0];} forEach ["place","edit","delete","destroy","group","synchronize"];
 
-// Check if F3 AI Skill Selector is active and assign corresponding event-handler
-if({!isNil _x} count ["f_param_AISkill_BLUFOR","f_param_AISkill_INDP","f_param_AISkill_OPFOR"] > 0) then {
-	_curator addEventHandler ['CuratorObjectPlaced',{
+// Add event-handler for setAISkill
+private _curatorEH = _curator getVariable ["f_curator_EH", -1];
+if (_curatorEH == -1) then {
+	_curatorEH = _curator addEventHandler ['CuratorObjectPlaced',{
 		params[
 			["_curator", objNull, [objNull]],
 			["_entity", objNull, [objNull]]
 		];
 		{
 			[_x] call f_fnc_setAISkill;
+
+			if(!isNil "f_var_disableThermals_enabled" && {f_var_disableThermals_enabled}) then {
+				_x disableTIEquipment true;
```

Comment:
This operates on the crew (which doesn't actually do anything AFAIK) rather than the vehicle.